From stable-bounces@linux.kernel.org Tue Aug 21 21:05:24 2007
Author: Chuck Ebbert <cebbert@redhat.com>
Date: Tue, 21 Aug 2007 21:05:14 -0700 (PDT)
Subject: Netfilter: Missing Kbuild entry for netfilter
To: stable@kernel.org
Cc: bunk@stusta.de
Message-ID: <20070821.210514.10298205.davem@davemloft.net>


Author: Chuck Ebbert <cebbert@redhat.com>

Add xt_statistic.h to the list of headers to install.

Apparently needed to build newer versions of iptables.

Signed-off-by: Chuck Ebbert <cebbert@redhat.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 include/linux/netfilter/Kbuild |    1 +
 1 file changed, 1 insertion(+)

--- a/include/linux/netfilter/Kbuild
+++ b/include/linux/netfilter/Kbuild
@@ -28,6 +28,7 @@ header-y += xt_policy.h
 header-y += xt_realm.h
 header-y += xt_sctp.h
 header-y += xt_state.h
+header-y += xt_statistic.h
 header-y += xt_string.h
 header-y += xt_tcpmss.h
 header-y += xt_tcpudp.h
